src/algos/value_iteration.jl

```diff
@@ -1,5 +1,17 @@
 using Optim
 
+
+"""
+#TODO: Is evaluate_policy the best name for this? Seems that we are evaluating the value function, or evaluating a current decision rule.
```

Comment:
The function name makes sense to me.

We are finding the value associated with a particular policy, or in other words, evaluating the policy.